tlsr9/drivers/B95/usbhw.c

```diff
@@ -31,8 +31,6 @@
 void usbhw_init(void)
 {
     pm_set_dig_module_power_switch(DIG_MODULE_USB, PM_POWER_UP);
-    pm_wait_dig_module_power_stable();

```

Comment:
Why is this function commented?

Reply:
in the new ble sdk , the func  pm_wait_dig_module_power_stable is disapperared .